Android/app/src/main/java/com/macmanus/jamie/bikerentalapp/model/dao/GithubUserDao.java

```diff
@@ -0,0 +1,20 @@
+package com.macmanus.jamie.bikerentalapp.model.dao;
```

Comment:
General note not related to this specific pull request but this path needs to be made generic